tools/container/runtime/containerd/containerd.go

```diff
@@ -80,11 +81,13 @@ func Flags(opts *Options) []cli.Flag {
 }
 
 // Setup updates a containerd configuration to include the nvidia-containerd-runtime and reloads it
-func Setup(c *cli.Context, o *container.Options, co *Options) error {
+func Setup(c *cli.Context, hostRoot string, o *container.Options, co *Options) error {
```

Comment:
I'd prefer:
```suggestion
func Setup(c *cli.Context, o *container.Options, co *Options, hostRoot string) error {
```
since this would make pulling this into other options later cleaner.